docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -67,6 +67,14 @@ There are two primary topologies within many event-driven architectures:
 
   Another challenge with asynchronous communication is data loss. If any of the components crashes before successfully processing and handing over the event to its next component, then the event is dropped and never makes it into the final destination. To minimize the chance of data loss, persist in-transit events and remove or dequeue the events only when the next component has acknowledged the receipt of the event. These features are usually known as _client acknowledge mode_ and _last participant support_.
 
+- Implementing a traditional request-response pattern.
+
+  Sometimes, the event producer requires an immediate response from the event consumer, such as obtaining a customer eligibility before proceeding with an order. In event-driven architecture, synchronous communication can be achieved through [request-response messaging](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html). This pattern can be implemented in through utilizing multiple queues - a request queue, and a response queue.
+
+  - **Establish two permanent queues**; a request queue and a reply queue. The event producer sends an asynchronous request to the request queue, pauses other operation on that task, and waits for a response in the reply queue. Event consumers then process the request and send the reply back through the preestablished reply queue. The approach utilizes a session ID for tracking so the event producer knows which message in the response queue is related to the specific request. This technique is a common implementation.
```

Comment:
> pauses other operation on that task

Just to be clear: My understanding is that event producer pauses all other operations, not just operations regarding that specific task.

Reply:
Why would it do that?  That seems highly inefficient and would be a terrible bottleneck.

Reply:
Fair point.